packages/devtools_app/lib/src/analytics/_analytics_web.dart

```diff
@@ -590,27 +588,29 @@ bool _userApplicationDimensionsComputed = false;
 Future<void> computeUserApplicationCustomGTagData() async {
   if (_userApplicationDimensionsComputed) return;
 
-  assert(serviceManager.connectedApp.isFlutterAppNow != null);
-  assert(serviceManager.connectedApp.isDartWebAppNow != null);
-  assert(serviceManager.connectedApp.isProfileBuildNow != null);
-
-  if (serviceManager.connectedApp.isFlutterAppNow) {
-    userPlatformType = (await serviceManager.service.isProtocolVersionSupported(
-            supportedVersion: SemanticVersion(major: 3, minor: 24)))
-        ? serviceManager.vm.operatingSystem
-        : 'unknown';
+  assert(serviceManager.connectedApp!.isFlutterAppNow != null);
+  assert(serviceManager.connectedApp!.isDartWebAppNow != null);
+  assert(serviceManager.connectedApp!.isProfileBuildNow != null);
+
+  const unknownOS = 'unknown';
+  if (serviceManager.connectedApp!.isFlutterAppNow!) {
+    userPlatformType = (await serviceManager.service!
+            .isProtocolVersionSupported(
+                supportedVersion: SemanticVersion(major: 3, minor: 24)))
+        ? serviceManager.vm!.operatingSystem ?? unknownOS
+        : unknownOS;
   }
-  if (serviceManager.connectedApp.isFlutterWebAppNow) {
+  if (serviceManager.connectedApp!.isFlutterWebAppNow) {
     userAppType = appTypeFlutterWeb;
-  } else if (serviceManager.connectedApp.isFlutterAppNow) {
+  } else if (serviceManager.connectedApp!.isFlutterAppNow!) {
```

Comment:
Should it be `.isFlutterAppNow == true` instead?

Reply:
we have an assert earlier in the method that checks that `serviceManager.connectedApp!.isFlutterAppNow != null`